Plugins/NativeCamera/Source/CameraDevice.h

```diff
@@ -95,14 +95,14 @@ namespace Babylon::Plugins
         arcana::task<CameraDimensions, std::exception_ptr> OpenAsync(const CameraTrack& track);
         void Close();
         CameraDimensions UpdateCameraTexture(bgfx::TextureHandle textureHandle);
-        TakePhotoTask TakePhoto(PhotoSettings photoSettings);
+        TakePhotoTask TakePhotoAsync(PhotoSettings photoSettings);
 
         const std::vector<CameraTrack>& SupportedResolutions() const;
         const std::vector<std::unique_ptr<Capability>>& Capabilities() const;
 
         // Gets high resolution photo capture capabilities for the currently opened stream/track.
-        Plugins::PhotoCapabilities PhotoCapabilities() const;
-        Plugins::PhotoSettings DefaultPhotoSettings() const;
+        const Plugins::PhotoCapabilities PhotoCapabilities() const;
```

Comment:
Still missing `&`
```suggestion
        const Plugins::PhotoCapabilities& PhotoCapabilities() const;
```

Reply:
Original intention was to just return a copy because they are small structs, but I will switch it to const refs.